_sass/_layout.scss

```diff
@@ -751,28 +778,4 @@ div.facebook {
         display: none;
     }
 }
-
 // Bootstrap Overwrites
-table.sortable > thead th:hover:not([data-defaultsort="disabled"]) {
```

Comment:
why is this block removed?

* This breaks the hover on the tables and we again have an unreadable color
* This breaks the menu on md devices